lib/utils/concatAddress/index.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * @example
- * // 0x123..456
+ * // 0x123…456
  */
 export const concatAddress = (address: string) => {
-  return address.slice(0, 5) + "..." + address.slice(address.length - 3);
+  return address.slice(0, 5) + "…" + address.slice(address.length - 3);
```

Comment:
this change is an a11y improvement since screen readers can properly recognize this whereas they might struggle with three full stop characters

Reply:
https://tosbourn.com/the-importance-of-using-the-ellipsis-character/